data/plugins/downloaders/EarthDownloader.py

```diff
@@ -18,29 +18,67 @@
 import os
 import subprocess
 
+from variety import _
+
 logger = logging.getLogger('variety')
 
-from variety import Downloader
+from variety.plugins.downloaders.SimpleDownloader import SimpleDownloader
+
 
-EARTH_IMAGE_URL = "https://static.die.net/earth/mercator/1600.jpg"
+EARTH_IMAGE_URL = "http://images.opentopia.com/sunlight/world_sunlight_map_rectangular.jpg"
```

Comment:
Repeated [from #173](https://github.com/varietywalls/variety/pull/173#issuecomment-493655586): Please do not use the “non Cloudflare’d mirror” URL because it is not a true mirror. It stamps some branding into the image that I categorically do not want to see on my desktop. Make sure you introduce a working user agent string of your choosing instead. Also the switch to the other URL causes the image to be less recent, as I have just seen by comparing https://static.die.net/earth/mercator/1600.jpg and http://images.opentopia.com/sunlight/world_sunlight_map_rectangular.jpg.

Reply:
Addressed, see https://github.com/varietywalls/variety/pull/173#issuecomment-493711328

Reply:
Thank you!